### ISSUE TITLE
King density profile implementation

### ISSUE THREAD

**Issue by ejullo** [GalSim-developers/GalSim#1134]
Hello 
I would like to simulate globular clusters with Galsim. They are usually modelled with a King surface brightness profile. Do you know if this SB is already implemented in Galsim. I couldn't find it. 
Eric

**Comment by rmjarvis** [comment]
No, it's not.  I've thought about adding a class that could take in an arbitrary user-provided function for the surface brightness profile.  This might be a good concrete use case to motivate that.

In the meanwhile, one way that might be workable for you is to use `InterpolatedImage`.  You can draw the King profile at some reasonable resolution and then use that class to treat the image as a surface brightness profile, which you can resize, shear, convolve by a PSF, etc.
